notebooks/254-llm-chatbot/254-rag-chatbot.ipynb

```diff
@@ -392,34 +391,29 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from converter import converters\n",
```

Comment:
why converters is removed ?

Reply:
I also noted that and fixed in another commit, it is just wrong copy-paste